tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -31,23 +31,35 @@ class StreamsBrokerDownResilience(BaseStreamsTest):
     message = "processed[0-9]*messages"
     connected_message = "Discovered group coordinator"
 
-    def __init__(self, test_context):
+    def __init__(self, test_context, kafka):
         super(StreamsBrokerDownResilience, self).__init__(test_context,
+                                                          kafka,
                                                           topics={self.inputTopic: {'partitions': 3, 'replication-factor': 1},
                                                                   self.outputTopic: {'partitions': 1, 'replication-factor': 1}},
                                                           num_brokers=1)
 
     def setUp(self):
         self.zk.start()
 
+    def prop_file(self):
+        properties = {streams_property.STATE_DIR: self.PERSISTENT_ROOT,
+                      streams_property.KAFKA_SERVERS: self.kafka.bootstrap_servers()}
+        if self.UPGRADE_FROM is not None:
```

Comment:
nit: the `StreamsBrokerDownResilience` doesn't need the `UPGRADE_FROM` or `UPGRADE_TO` variables.

Reply:
Will remove.